src/viam/services/navigation/navigation.py

```diff
@@ -22,36 +22,208 @@ class Navigation(ServiceBase):
 
     @abc.abstractmethod
     async def get_paths(self, *, timeout: Optional[float]) -> List[Path]:
+        """
+        Get each path, the series of geo points the robot plans to travel through
+        to get to a destination waypoint, in the machine’s motion planning.
+
+        ::
+
+            my_nav = NavigationClient.from_robot(robot=robot, name="my_nav_service")
+
+            # Get a list containing each path stored by the navigation service
+            paths = await my_nav.get_paths()
+
+        Args:
+            timeout (Optional[float]): An option to set how long to wait (in seconds)
+                before calling a time-out and closing the underlying RPC call.
+
+        Returns:
+            (List[navigation.Path]): An array comprised of Paths, each path being a user-provided destination,
+                or, Waypoint and the set of geopoints that the robot expects to travel through to get there.
```

Comment:
Can we rewrite this slightly? I'm a little confused by it. Something like `...each path being a either user-provided destination, or a Waypoint and the set of geopoints...` maybe?

Reply:
Looks great, thanks!